diff --git a/src/__main__.py b/src/__main__.py
index 4c6c70f..7db66e7 100644
--- a/src/__main__.py
+++ b/src/__main__.py
@@ -24,10 +24,12 @@ if __package__ is None and not hasattr(sys, 'frozen'):
     path = os.path.realpath(os.path.abspath(__file__))
     sys.path.insert(0, os.path.dirname(os.path.dirname(path)))
 
-if os.path.isdir("lib"):
-    sys.path.insert(0, os.path.abspath("lib"))
-elif os.path.isdir("../lib"):
-    sys.path.insert(0, os.path.abspath("../lib"))
+# 'lib' contains local executables, FAF overlay sets everything up in PATH
+if sys.platform == 'win32':
+    if os.path.isdir("lib"):
+        sys.path.insert(0, os.path.abspath("lib"))
+    elif os.path.isdir("../lib"):
+        sys.path.insert(0, os.path.abspath("../lib"))
 
 from PyQt4 import QtGui, uic
 
