diff --git a/src/__main__.py b/src/__main__.py
index ba702f7..3e495d3 100644
--- a/src/__main__.py
+++ b/src/__main__.py
@@ -50,6 +50,8 @@ else:
 import util
 if sys.platform == 'win32':
     util.COMMON_DIR = os.path.join(os.getcwd(), "res")
+else:
+    os.chdir(util.COMMON_DIR)
 
 # Set up crash reporting
 excepthook_original = sys.excepthook
diff --git a/src/util/__init__.py b/src/util/__init__.py
index 1a2276e..072046d 100644
--- a/src/util/__init__.py
+++ b/src/util/__init__.py
@@ -25,11 +25,17 @@ LOGFILE_MAX_SIZE = 256 * 1024  #256kb should be enough for anyone
 
 UNITS_PREVIEW_ROOT = "{}/faf/unitsDB/icons/big/".format(Settings.get('content/host'))
 
+# Allow for hardcoding res dir at install time
+UNIX_SHARE_DIR = ""
+
 # On Windows the res dir is relative to the executable or main.py script
 COMMON_DIR = os.path.join(os.getcwd(), "res")
 if sys.platform != 'win32' and not os.path.exists(COMMON_DIR):
-    #On Linux the res dir is installed as /usr/share/fafclient
-    COMMON_DIR = os.path.join("/usr", "share", "fafclient")
+    #On Linux the default res dir is installed as /usr/share/fafclient
+    if UNIX_SHARE_DIR == "":
+        COMMON_DIR = os.path.join("/usr", "share", "fafclient")
+    else:
+        COMMON_DIR = UNIX_SHARE_DIR
 
 # These directories are in Appdata (e.g. C:\ProgramData on some Win7 versions)
 if 'ALLUSERSPROFILE' in os.environ:
