diff --git a/src/__main__.py b/src/__main__.py
index e03ba86..af31f13 100644
--- a/src/__main__.py
+++ b/src/__main__.py
@@ -32,10 +32,12 @@ if __package__ is None and not hasattr(sys, 'frozen') and not imp.find_module("c
     path = os.path.realpath(os.path.abspath(fafclient.__file__))
     sys.path.insert(0, os.path.dirname(path))
 
-if os.path.isdir("lib"):
-    sys.path.insert(0, os.path.abspath("lib"))
-elif os.path.isdir("../lib"):
-    sys.path.insert(0, os.path.abspath("../lib"))
+# 'lib' contains local executables, FAF overlay sets everything up in PATH
+if sys.platform == 'win32':
+    if os.path.isdir("lib"):
+        sys.path.insert(0, os.path.abspath("lib"))
+    elif os.path.isdir("../lib"):
+        sys.path.insert(0, os.path.abspath("../lib"))
 
 from PyQt4 import QtGui, uic
 
